packages/x-components/src/x-modules/scroll/components/main-scroll.vue

```diff
@@ -55,7 +63,7 @@
      *
      * @public
      */
-    @Prop({ default: '' })
+    @Prop({ default: undefined })
     public margin!: string;
```

Comment:
👮 Types police officer here.
If you are going to leave the property optional, then you should flag it with `?` instead of `!` to ensure that the code that uses it knows that it can be `undefined`. Also, props are optional by default, so you don't need to specify the `default: undefined`.

```suggestion
    @Prop()
    public margin?: string;
```

Or, if you prefer @diegopf suggestion and avoid working with possible `undefined` values, this should work. Here you can still use the `!` operator because you are always guaranteeing to work with a `string`.

```suggestion
    @Prop({ default: '0px' })
    public margin!: string;
```